### PR TITLE
Delete API token on web console logout

### PR DESCRIPTION
**Pull request by liggitt**

### CODE REVIEW

**Comment by liggitt** [comment]
@jwforres review

File: assets/app/scripts/services/auth.js [hunk 1]
```diff
@@ -144,11 +155,39 @@ angular.module('openshiftConsole')
...
+        var user = userStore.getUser();
+        var token = userStore.getToken();
+        var wasLoggedIn = this.isLoggedIn();
+        _logoutPromise = logoutService.logout(user, token).then(function() {
+          if (debug) { console.log("Logout service success"); }
+        }).catch(function(err) {
+          if (debug) { console.log("Logout service error", err); }
+        }).finally(function() {
+          self.setUser(null, null);
+          var isLoggedIn = self.isLoggedIn();
```
Comment (jwforres) [inline assets/app/scripts/services/auth.js]: doesn't self.setUser(null, null) mean that self.isLoggedIn() would be false?

File: assets/app/scripts/services/data.js [hunk 2]
```diff
@@ -160,6 +160,55 @@ angular.module('openshiftConsole')
 // context:   API context (e.g. {project: "..."})
 // opts:      force - always request (default is false)
```
Comment (jwforres) [inline assets/app/scripts/services/data.js]: remove force

File: assets/app/scripts/services/auth.js (hunk 1 above)
Comment (liggitt) [inline assets/app/scripts/services/auth.js]: Added a comment explaining why

File: assets/app/scripts/services/data.js (hunk 2 above)
Comment (liggitt) [inline assets/app/scripts/services/data.js]: done

**Comment by liggitt** [comment]
comments addressed, ready for review/test

**Comment by jwforres** [comment]
[merge]

**Pull request merged**